### ISSUE TITLE
Unable to read `wsUri` 

### ISSUE THREAD

**Issue by bredamatt** [paritytech/zombienet#1115]
### Issue Description

I am using a `js-script` call in a `.zndsl` file, where I am trying to get the `wsUri` from the `networkInfo.nodesByName[nodeName].wsUri`, but somehow this isn't working for the collator_groups in my network configuration.

What I am trying to do is to send transfers from each collator in the collator_group in parallel to make sure all the collations are full of transfers to maximize the sTPS metric, hence the `wsUri` needs to be defined for each node in the group when initializing a process in parallel on a node group.

### Steps to reproduce the issue

I am using the current `.zndsl` file and the test that fails is the 
```
# Sending the extrinsics
stps: js-script ../utils.js with "send_balance_transfers,{{SENDERS}},5000,polkadot-parachain," within 200 seconds
```
step.

Here is the file:

```
Network: ./single-para-native.json
Creds: config

alice: is up within 600 secs
bob: is up within 600 secs
collator01: is up within 600 secs

# Check progress
alice: parachain 1000 is registered within 60 seconds
# alice: parachain 1000 block height is at least 10 within 300 seconds

# Setting sender indexes
{% for i in range(0, SENDERS) -%}
  stps-{{ i }}: run ../set_sender_index.sh with "{{i}}" within 2 seconds
{%- endfor %}

# Sending the extrinsics
stps: js-script ../utils.js with "send_balance_transfers,{{SENDERS}},5000,polkadot-parachain," within 200 seconds

# Start the TPS scraper on the validator, args: <cmd>, <num_senders>, <expected_trx>, <runtime>, <collator_to_send_to>, <para_id>
alice: js-script ../utils.js with "calculate_tps,{{SENDERS}},5000,polkadot-parachain,collator01,1000" within 600 secs
```

Where the javascript file that checks for `wsUri` looks like this:

```
const wsUri = networkInfo.nodesByName[nodeName].wsUri;
```

This means it should be able to find the `wsUri` and in fact, it does when I print it to the console. Also it works in a different test running on `alice`, (i.e. the `calculate_tps` test below the `send_balance_transfers`), but not in the parallel step. 

The network config I use is this:

```
{
  "settings": {
    "timeout": 1000
  },
  "relaychain": {
    "default_image": "docker.io/parity/polkadot:latest",
    "chain": "rococo-local",
    "default_command": "polkadot",
    "nodes": [
      {
        "name": "alice",
        "validator": true,
          "ws_port": 9944,
          "args": [
            "--alice",
            "-lparachain=debug",
            "--pool-limit=1000000000",
            "--pool-kbytes=10000000000"
          ]
      },
      {
        "name": "bob",
        "validator": true,
        "ws_port": 9944,
        "args": [
          "--alice",
          "-lparachain=debug",
          "--pool-limit=1000000000",
          "--pool-kbytes=10000000000"
        ]
      }
    ]
  },
  "parachains": [
    {
      "id": 1000,
      "cumulus_based": true,
      "genesis": {
        "runtime": {
          "balances": {
            "balances": {% include "../funded-accounts.json" %}
          }
        }
      },
      "collator": {
        "name": "collator01",
        "ws_port": "9944",
        "image": "docker.io/parity/polkadot-parachain",
        "command": "polkadot-parachain",
        "args": [
          "-lparachain=debug",
          "--pool-limit=1000000000",
          "--pool-kbytes=10000000000"
        ]
      },
      "collator_groups": [
        {
          "name": "stps",
          "ws_port": "9944",
          "args": [
            "-lparachain=debug",
            "--pool-limit=1000000000",
            "--pool-kbytes=10000000000"
          ],
          "count": "{{SENDERS}}"
        }
      ]
    }
  ]
}
```

### Describe the results you received

It returns the `Unable to read `wsUri`  error for the test that I need to complete.

### Describe the results you expected

it should be able to resolve the `wsUri`.

### Zombienet version

The latest released version

### Provider

Native

### Provider version

```yaml
## For binaries
polkadot 0.9.41-ee8aa322842
polkadot-parachain 0.9.420-31b1469d830
```

### Upstream Latest Release

Yes

### Additional environment details

SENDERS=2

**Comment by bredamatt** [comment]
This makes sense since it works for the `alice` node, but not for the `stps` collator group! Thanks @pepoviola!

**Comment by pepoviola** [comment]
Fixed by #1116 . Thanks!

**Comment by bredamatt** [comment]
I found a workaround which solved the problem so closing the issue
